inst/pages/11_taxonomic_information.qmd

```diff
@@ -336,22 +336,22 @@ references.
    in microbiome-based classification performance [@Giliberti2022,
    Karwowska2024].
 
- * 'z' Z transformation scales data to zero mean and unit variance;
-   this is used to bring features (or samples) to more comparable
+ * 'z' Z transformation scales data to zero mean and unit variance.
```

Comment:
This should be updated to "scale" (we renamed it recently).

Reply:
Sorry, in fact it is "standardize" (and not "scale") I think. In the latest mia version.